nires/removenoise.py

```diff
@@ -9,12 +9,10 @@
 from astropy.io import fits
 #title = ''
 
-#title,prefix=imD.returnInst(sys.argv[1])
-name=imD.nameResolve(sys.argv[2],'s*')
+#print sys.argv[2]
+#name=imD.nameResolve(sys.argv[2],'s*')
 
-
-
-noise_image = fits.open(name)
+noise_image = fits.open('s151112_a000017.fits')
```

Comment:
Lets make this code generalized, so the user can input a file rather than hard coding